sdk/examples/intkey_go/src/sawtooth_intkey/handler/handler.go

```diff
@@ -4,12 +4,16 @@ import (
 	"crypto/sha512"
 	"encoding/hex"
 	"fmt"
+    "os"
```

Comment:
Looks like spaces here, we probably need gofmt on these files.  We need to figure out how to add this to run_lint.